services/docker/templates/compose.go

```diff
@@ -203,20 +204,35 @@ func TokaidoDockerSiteVolumeAttach(path string) []byte {
 		log.Fatalf("Could not resolve your home directory: %v", err)
 	}
 
-	return []byte(`services:
-  drush:
-    volumes:
-      - ` + path + `:/tokaido/site
-      - ` + h + `/.gitconfig:/home/tok/.gitconfig
-      - ` + h + `/.drush:/home/tok/.drush
-      - tok_composer_cache:/home/tok/.composer/cache
+	vols := `services:
   nginx:
     volumes:
       - ` + path + `:/tokaido/site
   fpm:
     volumes:
       - ` + path + `:/tokaido/site
-`)
+  drush:
+    volumes:
+      - ` + path + `:/tokaido/site
+      - tok_composer_cache:/home/tok/.composer/cache`
+
+	// We'll mount the .gitconfig and .drush paths if they exist
+	gp := h + "/.gitconfig"
+	dp := h + "/.drush:/home/tok/.drush"
+
+	_, err = os.Stat(gp)
+	if err == nil {
+		vols = vols + `
+      - ` + h + `/.gitconfig:/home/tok/.gitconfig`
+	}
+
+	_, err = os.Stat(dp)
+	if err == nil {
+		vols = vols + `
+      - ` + h + `/.gitconfig:/home/tok/.drush`
+	}
```

Comment:
For checking existence for dirs/files, I created an function in the `fs` package

```
// CheckExists - Check if a file exists or not
func CheckExists(path string) bool {
	_, err := os.Stat(path)
	if os.IsNotExist(err) {
		return false
	}

	return true
}
```

 Difference is that it runs `os.IsNotExist` which has a pretty nuanced difference

https://stackoverflow.com/questions/12518876/how-to-check-if-a-file-exists-in-go#12518877

Reply:
thanks! I've updated accordingly